### ISSUE TITLE
Not running on Ubuntu 22.04

### ISSUE THREAD

**Issue by eroszanchetta** [FrancescoCeruti/linux-show-player#245]
The application doesn't start in Ubuntu 22.04.

Launching it from the command line results in the following error:

ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)

I tried reverting to Python 3.8 (default in Ubuntu 22.04 is Python 3.10), but I got another error:

ModuleNotFoundError: No module named 'PyQt5.sip'

At this point I gave up.

**Comment by JonnyTech** [comment]
Are you using the AppImage?

Else, according to https://github.com/FrancescoCeruti/linux-show-player/wiki/Install toy need to install lots of dependencies including `python3-pyqt5`.

**Comment by eroszanchetta** [comment]
No, I used the version that came with the distro, i.e. I installed it using

apt-get install linux-show-player

**Comment by JonnyTech** [comment]
Looks like the last releases AppImage was 0.5.2: https://github.com/FrancescoCeruti/linux-show-player/releases/download/v0.5.2/LinuxShowPlayer-v0.5.2-x86_64.AppImage

@FrancescoCeruti can you release a newer build please?

**Comment by s0600204** [comment]
A downstream bug report has been filed, and may be found here: https://bugs.launchpad.net/ubuntu/+source/linux-show-player/+bug/1987073
